packages/framework/src/Model/Product/ProductFacade.php

```diff
@@ -442,4 +480,15 @@ public function getOneByCatnumExcludeMainVariants($productCatnum)
     {
         return $this->productRepository->getOneByCatnumExcludeMainVariants($productCatnum);
     }
+
+    /**
+     * @internal Will be removed in the next major release
+     * @deprecated
+     */
+    protected function validateInjectedDependencies(): void
```

Comment:
Just wondering... should this be protected or private? :)

Reply:
According to our PrivateVisibilityFixer, it should be protected as in `Model` namespace is `private` forbidden